packages/goflow2/data_stream/sflow/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,269 @@
+---
+description: Process goflow2 sflow data
+processors:
+- set:
+    field: event.kind
+    value: event
+    tag: '1'
+- append:
+    field: event.category
+    value:
+    - network
+    tag: '2'
+- append:
+    field: event.type
+    value:
+    - connection
+    tag: '3'
+- set:
+    field: ecs.version
+    value: 8.11.0
+    tag: '4'
+- json:
+    field: message
+    add_to_root: true
```

```suggestion
    target_field: goflow2
```

Extract the JSON fields to a vendor field instead (`goflow2`). The ECS fields can still be renamed below.

---

fxied in https://github.com/elastic/integrations/pull/10561/commits/2130f6c0ff90203c891e3538844fe21a7e3bc161